pkgs/applications/video/mpv/scripts/acompressor.nix

```
@@ -0,0 +1,20 @@
+{ stdenvNoCC, mpv-unwrapped, lib }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mpv-acompressor";
+  version = mpv-unwrapped.version;
+  src = "${mpv-unwrapped.src.outPath}/TOOLS/lua/acompressor.lua";
+  dontBuild = true;
+  dontUnpack = true;
+  installPhase = ''
+    install -Dm644 ${src} $out/share/mpv/scripts/acompressor.lua
+  '';
+  passthru.scriptName = "acompressor.lua";
```

```suggestion
stdenvNoCC.mkDerivation rec {
  pname = "mpv-acompressor";
  version = mpv-unwrapped.version;

  src = "${mpv-unwrapped.src.outPath}/TOOLS/lua/acompressor.lua";

  dontBuild = true;
  dontUnpack = true;

  installPhase = ''
    install -Dm644 ${src} $out/share/mpv/scripts/acompressor.lua
  '';

  passthru.scriptName = "acompressor.lua";
```

new lines make it more readable and are meant to reduce conflicts (esp the function arguments/dependencies)